tests/backbeat/tests/api/metrics.js

```diff
@@ -1,191 +1,202 @@
+                    },
+                };
+                getAndCheckResponse(`${pathPrefix}/all/backlog`, expectedBody,
+                    next);
+            },
+            next => makeGETRequest(`${pathPrefix}/all`, (err, res) => {
+                assert.ifError(err);
+                getResponseBody(res, (err, body) => {
                     assert.ifError(err);
-                    versionId = res.VersionId;
+                    prevCompletions = body.completions.results;
+                    prevThroughput = body.throughput.results;
                     next();
-                }),
-            next => scalityUtils.compareObjectsAWS(srcBucket, destBucket, key,
-                undefined, next),
-            next => makeDelayedGETRequest('/_/backbeat/api/metrics/crr/' +
-                `${destLocation}/throughput/${destBucket}/${destinationKey}` +
-                `?versionId=${versionId}`,
-                (err, res) => {
-                    assert.ifError(err);
-                    assert.equal(res.statusCode, 200);
-
-                    getResponseBody(res, (err, body) => {
-                        assert.ifError(err);
-                        assert(body.description);
-                        assert(body.throughput);
-
-                        next();
-                    });
-                }),
+                });
+            }),
+            // NOTE: metrics are doubled because 2 destination locations
+            next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(100),
+                next),
+            next => scalityUtils.waitUntilReplicated(srcBucket, key, undefined,
+                next),
+            next => {
+                const expectedBody = {
+                    completions: {
+                        description: 'Number of completed replication ' +
+                        'operations (count) and number of bytes transferred ' +
+                        '(size) in the last 900 seconds',
+                        results: {
+                            count: prevCompletions.count + 2,
+                            size: prevCompletions.size + 200,
+                        },
+                    },
+                };
+                // Just make sure completions has been incremented. We know if
+                // completions matches, all other data should be as well
+                getAndCheckResponse(`${pathPrefix}/all/completions`,
+                    expectedBody, next);
+            },
+            next => {
+                const expectedBody = {
+                    throughput: {
+                        description: 'Current throughput for replication ' +
+                        'operations in ops/sec (count) and bytes/sec (size) ' +
+                        'in the last 900 seconds',
+                        results: {
+                            count: (parseFloat(prevThroughput.count) +
+                                (2 / 900)).toFixed(2),
```

Comment:
Minor: looks like this will generally be `'0.00'`, maybe we could consider putting N objects to the point that the value is increased in the API response.

Reply:
do you mean replicate N objects and each time check that throughput increases?

Reply:
Was thinking that replicating >= 5 objects would result in the API responding with a floating point > `0.00`. It's possibly overkill since having the `size` property increase probably suffices.